docs/api-docs/channels/channels-overview.md

```diff
@@ -119,7 +119,7 @@ Examples:
 
 - Amazon
 - eBay
-- Facebook Marketplace
+- Meta Marketplace
```

Comment:
let's verify that this is called `Meta Marketplace` for sure. I think they may have rebranded that.